common-service/src/main/java/org/sunbird/global/ServiceBaseGlobal.java

```diff
@@ -0,0 +1,271 @@
+package org.sunbird.global;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.BadgingJsonKey;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.common.models.util.ProjectUtil.Environment;
+import org.sunbird.common.request.ExecutionContext;
+import org.sunbird.common.request.HeaderParam;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.controllers.BaseController;
+import org.sunbird.telemetry.util.TelemetryUtil;
+import org.sunbird.util.CassandraStartUpUtil;
+import org.sunbird.util.authentication.RequestInterceptor;
+import play.Application;
+import play.libs.F.Promise;
+import play.libs.Json;
+import play.mvc.Action;
+import play.mvc.Http;
+import play.mvc.Http.Context;
+import play.mvc.Http.Request;
+import play.mvc.Result;
+import play.mvc.Results;
+
+/**
+ * This class will work as a filter. All play request pass through this filter onCall method. Class
```

Comment:
Base class for sunbird services which require Cassandra connection and authentication of API requests to perform specific tasks at application start up or shut down.